### PR TITLE
Switch Helium and Hydrogen to proper naming

### PR DESCRIPTION
**Pull request by nothingneko**
Switch `tau-helium` and `tau-hydrogen` to `helium-gtk-theme` and `hydrogen-icon-theme`

### CODE REVIEW

File: anda/themes/helium-gtk-theme/anda.hcl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project pkg {
+	arches = ["x86_64", "aarch64"]
```
Comment (lleyton) [inline anda/themes/helium-gtk-theme/anda.hcl]: why? It's noarch

**Review by lleyton — CHANGES_REQUESTED** [review]

File: anda/themes/helium-gtk-theme/anda.hcl (hunk 1 above)
Comment (nothingneko) [reply anda/themes/helium-gtk-theme/anda.hcl]: it was marked as x86 only, I assumed this was the way to do it

Comment (nothingneko) [reply anda/themes/helium-gtk-theme/anda.hcl]: just checked, fluent is the same setup?

Comment (lleyton) [reply anda/themes/helium-gtk-theme/anda.hcl]: where does it say x86 only?

Comment (nothingneko) [reply anda/themes/helium-gtk-theme/anda.hcl]: anda.hcl

Comment (madonuko) [reply anda/themes/helium-gtk-theme/anda.hcl]: If this is noarch, both builders will build noarch too?

File: anda/themes/helium-gtk-theme/helium-gtk-theme.spec [hunk 2]
```diff
@@ -12,6 +12,8 @@ BuildRequires:  sass
 BuildRequires:  meson
 BuildRequires:  ninja-build
 BuildRequires:  fdupes
+Provides: tau-helium = %{version}-%{release}
+Obsoletes: tau-helium < 1.8.23-1
```
Comment (madonuko) [inline anda/themes/helium-gtk-theme/helium-gtk-theme.spec]:
```suggestion
Obsoletes: tau-helium < 1.8.23-2
```

**Review by madonuko — CHANGES_REQUESTED** [review]

File: anda/themes/helium-gtk-theme/anda.hcl (hunk 1 above)
Comment (lleyton) [reply anda/themes/helium-gtk-theme/anda.hcl]: thus there's no reason to have it build on multiple machines

Comment (nothingneko) [reply anda/themes/helium-gtk-theme/anda.hcl]: i misunderstood then, lemme fix this

Comment (nothingneko) [reply anda/themes/helium-gtk-theme/anda.hcl]: we may want to document that that's the procedure for noarch

**Comment by lleyton** [comment]
fix the failing build, I recommend you try to build things locally so you know if they work or not

**Comment by lleyton** [comment]
also, search for instances of references to the old package names and update them

**Comment by nothingneko** [comment]
> also, search for instances of references to the old package names and update them

do you want this in this pr or?

**Comment by lleyton** [comment]
yes, this PR

File: anda/lib/libhelium/libhelium.spec [hunk 3]
```diff
@@ -22,8 +22,8 @@ BuildRequires:  pkgconfig(gtk4) >= 4.4
 Requires: gtk4 >= 4.4
 Requires: glib2 >= 2.66.0
 Requires: libgee >= 0.20
-Requires: tau-helium >= 1.1.25
-Requires: tau-hydrogen
+Requires: helium-gtk-theme >= 1.1.25
```
Comment (lleyton) [inline anda/lib/libhelium/libhelium.spec]: maybe remove the version constraint here IMO

**Comment by lleyton** [comment]
bump the libhelium release

File: anda/lib/libhelium/libhelium.spec (hunk 3 above)
Comment (nothingneko) [reply anda/lib/libhelium/libhelium.spec]: that's just the way it was before

**Review by lleyton — APPROVED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2064)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2065)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2066)|
|❌|el9|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
backport --pr 2058
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10825401253) for details

**Pull request merged**